Formula/abi-compliance-checker.rb

```diff
@@ -3,6 +3,7 @@ class AbiComplianceChecker < Formula
   homepage "http://ispras.linuxbase.org/index.php/ABI_compliance_checker"
   url "https://github.com/lvc/abi-compliance-checker/archive/1.99.9.tar.gz"
   sha256 "52b0daca89fcda73cde126497c8015ca823417074ba02fcff68b7acf2f45e516"
+  revision 1
```

Comment:
IIRC the test needs rewriting to account for GCC switching to single-digit versioning, `gcc-5` instead of `gcc-4.9` for example. This has been a problem for a while but has been stuck on various to-do lists.